### PR TITLE
Show tooltips for Idea Hub icon buttons.

### PR DESCRIPTION
**Pull request by kuasha420**
## Summary

Addresses issue #3906

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/Idea.js [hunk 1]
```diff
@@ -186,6 +191,10 @@ export default function Idea( props ) {
 									activity === IDEA_HUB_ACTIVITY_IS_PROCESSING
 								}
 								icon={ <UnpinIcon /> }
+								title={ __(
+									'Remove from saved"',
```
Comment (aaemnnosttv) [inline assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/Idea.js:195]: Oops!
```suggestion
									'Remove from saved',
```

**Review by aaemnnosttv — COMMENTED** [review]
Looks good, just the one little typo which I'll apply the suggestion for 👍

**Review by aaemnnosttv — APPROVED** [review]
LGTM 👍

File: assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/Idea.js (hunk 1 above)
Comment (kuasha420) [reply assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/Idea.js]: i-hang-my-head-in-shame

**Pull request merged**